R/build-home.R

```diff
@@ -261,14 +261,15 @@ cran_mirror <- function() {
     cran
   }
 }
-on_cran <- function(pkg) {
+on_cran <- function(pkg, cran = NULL) {
```

Comment:
Just set the default argument to `cran_mirror()`

BTW you shouldn't use `ifelse()` like this — it's exclusively for vectors. Instead you can use an inline if statement:

```R
cran <- if (is.null(cran)) cran_mirror() else cran
```

But this is such a common pattern most of my packages provide `%||%` so you can write:

```R
cran <- cran %||% cran_mirror()
```

But here I think it's fine to just set the default argument.